about.html

```diff
@@ -7,59 +7,103 @@
   <title>Sprint Challenge - About</title>
 
   <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
-  <link rel="stylesheet" href="css/index.css">
+  <link rel="stylesheet" href="css/about.css">
 
 </head>
 
 <body>
-    <div class="container about-page">
-     
-    About
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-
-
-    <img src="img/about-plan.png" alt="strategy">
-    
-    Strategy
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
-
-    <img src="img/about-working.png" alt="strategy">
-    
-    How We Work
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
-
-    <img src="img/about-office.png" alt="strategy">
-    
-    Places We Work
-
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
+    <div class="container-about-page">
+      <header>
+        <div class="head-nav">
+          <div class="flex-logo">
+              <img src="/web-Sprint-Challenge-User-Interface-and-Git/img/lambda-black.png" alt="Lambda logo">
+          </div>
+          <nav class="flex">
+              <a href="index.html">Home</a>
+              <a href="about.html">About</a>
+              <a href="#">Products</a>
+              <a href="#">Blog</a>
+              <a href="#">Contact</a>
+          </nav>
+         </div>
+          <div id="city-pic">
+            <img src="/web-Sprint-Challenge-User-Interface-and-Git/img/jumbo-about.png" alt="Picture of a city">
+          </div>
```

Comment:
There are places in the content area where you can use semantic HTML like `<section>` or `<article>`